src/experiments/preferences/api.js

```diff
@@ -35,6 +35,8 @@ ExtensionPreferencesManager.addSetting("dohRollout.state", {
       prefs[TRR_MODE_PREF] = 0;
       break;
     case "manuallyDisabled":
+      prefs[TRR_MODE_PREF] = 0;
```

Comment:
This is the default now anyway from line 29, maybe we should remove 35 too.

Reply:
I think this has to remain to toggle DoH off during temporary situations (like safe browsing) and back on non-safe browsing networks).

Reply:
Line 29 sets this to 0 anyway is my point, I think it doesn't do anything no?

Reply:
Ahh, I see it now. Removing both lines!